test/testers.jl

```diff
@@ -57,6 +57,15 @@ end
 abstract type MySpecialTrait end
 struct MySpecialConfig <: RuleConfig{Union{MySpecialTrait}} end
 
+my_id(x) = x
```

Comment:
This should be moved inside the `@testset`.
So it is clear what it is for testing.

Reply:
oh right, it's only the structs that have to be top level in 1.0